src/main/java/com/TableTalk/Enterprise/controllers/TableTalkController.java

```diff
@@ -1,168 +1,119 @@
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
+
 import java.io.IOException;
 import java.util.List;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.*;
 
 
 @Controller
-    public class TableTalkController {
-
+public class TableTalkController {
 
     @Autowired
     ITableTalkService TableTalkService;
-        /**
-         * Handle the root (/) endpoint and return a start page.
-         * @return
-         */
 
+    /**
+     * Handle the root (/) endpoint and return a start page.
+     *
+     * @return
+     */
     @RequestMapping("/")
     public String index() {
         return "start";
     }
 
     @GetMapping("/Game")
-
-    public ResponseEntity fetchGames(){
-
+    public ResponseEntity fetchGames() {
         return new ResponseEntity(HttpStatus.OK);
-
     }
 
-    @PostMapping(value="/Game", consumes="application/jason", produces="application/json")
-
-    public com.TableTalk.Enterprise.dto.Game createGame(@RequestBody com.TableTalk.Enterprise.dto.Game game){
-
+    @PostMapping(value = "/Game", consumes = "application/json", produces = "application/json")
+    public com.TableTalk.Enterprise.dto.Game createGame(@RequestBody Game game) {
         return game;
-
     }
 
     @DeleteMapping("/Game")
-
-    public ResponseEntity deleteGames(){
-
+    public ResponseEntity deleteGames() {
         return new ResponseEntity(HttpStatus.OK);
-
     }
 
-
-
     @GetMapping("/ProfilePicture")
-
-    public ResponseEntity fetchProfilePicture(){
-
+    public ResponseEntity fetchProfilePicture() {
         return new ResponseEntity(HttpStatus.OK);
-
     }
 
-    @PostMapping(value="/ProfilePicture", consumes="application/jason", produces="application/json")
-
-    public com.TableTalk.Enterprise.dto.ProfilePicture createProfilePicture(@RequestBody com.TableTalk.Enterprise.dto.ProfilePicture profilePicture){
-
+    @PostMapping(value = "/ProfilePicture", consumes = "application/json", produces = "application/json")
+    public com.TableTalk.Enterprise.dto.ProfilePicture createProfilePicture(@RequestBody ProfilePicture profilePicture) {
         return profilePicture;
-
     }
 
     @DeleteMapping("/ProfilePicture")
-
-    public ResponseEntity deleteProfilePicture(){
-
+    public ResponseEntity deleteProfilePicture() {
         return new ResponseEntity(HttpStatus.OK);
-
     }
 
-
-
     @GetMapping("/Room")
-
-    public ResponseEntity fetchRooms(){
-
+    public ResponseEntity fetchRooms() {
         return new ResponseEntity(HttpStatus.OK);
-
     }
 
-    @PostMapping(value="/Room", consumes="application/jason", produces="application/json")
-
-    public com.TableTalk.Enterprise.dto.Room createRoom(@RequestBody com.TableTalk.Enterprise.dto.Room room){
```

Comment:
Here is a technical suggestion: don't use full class paths, just import them.

Reply:
Good suggestion... so why didn't you do that in the code review branch?